### ISSUE TITLE
Lexicon files that contain entries with and without POS tag information

### ISSUE THREAD

**Issue by apmoore1** [UCREL/Multilingual-USAS#10]
The Swedish semantic lexicon contains 1860 entries that have no POS tag information, [example can be seen at this link](https://github.com/UCREL/Multilingual-USAS/blob/ef4ba7c33731af46740f4a2d8861368fa0207903/Swedish/semantic_lexicon_se.txt#L11814), whereby the majority of the entries do have POS tag information. I was wondering how we should handle this:
1. Allow POS tag information to be optional when other entries do have POS tag information
2. Have two files which contains only entries with POS tag information and another without.

I think the first option might be best as it reduces the number of files. Thoughts and suggestions are welcome :slightly_smiling_face:

**Comment by perayson** [comment]
Agreed, option one seems best. In that way, those line have implicit POS = wildcard. For Swedish, if the collaborators there are still interested, I can go back to retrace our steps and work out why those lines have no POS. They really ought to include one ideally, but if I remember correctly for some languages (not sure if this is true for Swedish) that once you have the base form, then it is not grammatically ambiguous.

**Comment by apmoore1** [comment]
From what I can see the Swedish single word lexicon is the only lexicon that has this implicit POS feature. Other lexicons that use the wildcard feature is, in both case for the MWE lexicon:
1. Welsh - 2 times
2. Russian - 1772 times

I agree option 1 is best. I might make a note in the [CONTRIBUTING file](https://github.com/UCREL/Multilingual-USAS/blob/master/CONTRIBUTING.md) that states this, e.g. if a POS tag cannot be assigned use the wildcard symbol `*`.

**Comment by perayson** [comment]
Agreed, and the inconsistency has arisen since we didn't enforce a common format previously I guess.
